devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -136,7 +136,7 @@ def parse(String description) {
                 }
                 break;
             case 0xFC00:
-                if (description?.startsWith('read attr -'))
+                if (description ? .startsWith('read attr -'))
```

Comment:
I just checked this in the interpreter to be sure, but this won't run.

Interestingly, `description ?. startsWith...` does, but we'd prefer `description?.startsWith`

Reply:
I have now fixed that. Must've happened when I was using a code beautifier to fix the indentation.